libs/test/SmartStoreTest/src/com/salesforce/androidsdk/store/SmartSqlExternalStorageTest.java

```diff
@@ -57,7 +57,7 @@
 	private static final String DEPARTMENTS_SOUP = "departments";
 
 	@Override
-	protected String getPasscode() {
+	protected String getEncryptionKey() {
```

Comment:
Unrelated to the config work - but thought that we should rename that method (getPasscode was a bit misleading).

Reply:
The encryption key is now automatically generated and applied internally by `SalesforceSDK`. This seems like a legacy artifact. Should we get rid of this interface (`getPasscode()`) to better represent the current design?